public/js/buildGherkin.js

```diff
@@ -340,7 +344,7 @@ function buildTable(testEl) { // el should be the test element
         });
         // add in any new values that may be missing
         for ( var key in variables) {
-            if ($.inArray(key, headerVals) == -1) {
+            if ($.inArray(key, headerVals) === -1) {
                 header.append("<th inheritFrom='" + variables[key] + "'>" + key + "</th>");
                 example.children('table').children('tbody').children('tr').each(function() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define this function outside of a loop. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=javascript%3AFunctionDefinitionInsideLoop)